src/rapids_dependency_file_generator/rapids_dependency_file_generator.py

```diff
@@ -112,8 +112,8 @@ def should_use_specific_entry(matrix_combo, specific_entry_matrix):
     return True
 
 
-def main(config_file, files):
-    with open(config_file, "r") as f:
+def main(config_file_path, files={}):
+    with open(config_file_path, "r") as f:
```

Comment:
Mode `"r"` is the default. The formatting/linting tool `pyupgrade` would remove this automatically.
```suggestion
    with open(config_file_path) as f:
```